controllers/utils.go

```diff
@@ -188,3 +198,27 @@ func compareMaps(m1, m2 map[string][]byte) bool {
 
 	return true
 }
+
```

Comment:
I would be very weary of rolling our own password generating code (even if it is a first time pass for admin). Maybe something simpler like:
```
func GenerateRandomPassword(length int) (string, error) {
    bytes := make([]byte, length)
    _, err := rand.Read(bytes)
    if err != nil {
        return "", err
    }
    return base64.URLEncoding.EncodeToString(bytes), nil
}
```

Also in this case we really want to use `crypto/rand` and not `math/rand`